package.json

```diff
@@ -32,7 +32,8 @@
     "mainClass": "XCUITestDriver",
     "scripts": {
       "build-wda": "./scripts/build-wda.js",
-      "open-wda": "./scripts/open-wda.js"
+      "open-wda": "./scripts/open-wda.js",
+      "tunnel-creation": "./scripts/tunnel-creation.mjs"
```

Comment:
I believe we need to also create a separate document to describe how to use this script